.github/scalardb-schema-loader.properties

```diff
@@ -0,0 +1,4 @@
+scalar.db.contact_points=scalardl-samples-cassandra-1
+scalar.db.contact_port=9042
+scalar.db.username=cassandra
+scalar.db.password=cassandra
```

Comment:
This file is added for fixing the end-to-end test.
We used C* CQL file to create a custom schema for testing before but now I think using scalardb-schema-loader might be properer.